src/main/java/org/apache/commons/compress/compressors/gzip/GzipParameters.java

```diff
@@ -432,6 +433,15 @@ public OS getOS() {
         return operatingSystem;
     }
 
+    /**
+     * Returns if the header crc is to be added (when writing) or was present (when reading).
+     *
+     * @return true is header crc si to be added (on write) or was found (after read).
+     */
```

Comment:
- This is not a good public API name IMO. "Hcrc" -> "HeaderCrc()". 
- New public and protected elements need Javadoc `@since` tags.